utils/utils.js

```diff
@@ -294,43 +294,63 @@ function findAllEnabledDestinations(
   if (sdkSuppliedIntegrations.All !== undefined) {
     allValue = sdkSuppliedIntegrations.All;
   }
-  const intgData = {};
```

Comment:
A better solution would be to convert this into an array,

```
intgData = [
  {
    key: intg_name_1,
    value: <incoming intg object>
  },
  {
    key: intg_name_2,
    value: <incoming intg object>
  }
...
]
```
as the whole point of this refactoring was to avoid redundant code.

Reply:
What benefit will I get by doing this?
For object 
intgData = [{key: intg_name_1, value: <incoming intg object>}] is same as `configPlaneEnabledIntegrations`

Reply:
Addressed